cluster-install/install.sh

```diff
@@ -94,6 +94,11 @@ readonly __WARN=30
 readonly __ERROR=40
 readonly __FATAL=50
 
+# __ERR_FILE should be used when wanting to capture stdout and stderr output of a command separately.
+# You can then use `get_captured_err` to get the captured error. Example:
+#     captured_stdout="$(curl -sS https://bad-url.com 2> "$_ERR_FILE")"
+#     captured_stderr="$(get_captured_err)"
+readonly __ERR_FILE="tmp_last_captured_error.txt"
```

Comment:
From what I could tell, there's no other way to separately capture both stdout/stderr without sending at least one into a file. 

We can generally get away with ignoring **stderr** and instead just check the return code or check if the returned output is empty, but in some cases, the **stderr** output is essential to figuring out what the hell is going on (most usages of curl is one of those cases).  This `__ERR_FILE` is for those situations, and the reason why it's a variable is mainly for consistency across the script.